src/components/ChatRoom.js

```diff
@@ -57,14 +57,33 @@ export default function ChatRoom() {
       });
   }, []);
 
+  const handleDelete = (id) => {
+    const token = localStorageService.get("auth-token");
+    let config = {
+      method: "delete",
+      maxBodyLength: Infinity,
+      url: BaseUrl + "chat/chatroom/" + id + "/",
+      headers: {
+        Authorization: `token ${token}`,
+      },
+    };
+    axios.request(config)
+    .then(() => {
```

Comment:
Upon deleted successfully, we need to refresh the list of chatrooms so the deleted chatroom does not show anymore.